build/Common.props

```diff
@@ -21,15 +21,22 @@
       Please sort alphabetically.
       Refer to https://docs.microsoft.com/en-us/nuget/concepts/package-versioning for semver syntax.
     -->
+    <CoverletCollectorPkgVer>[1.3.0,2.0)</CoverletCollectorPkgVer>
```

Comment:
Is this package going to be used by the release?
If not, move them to https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/build/Common.nonprod.props.

Reply:
Ah I forgot to check this file. I think I included duplicates since some package versions were already specified here. There should be no duplicate/shared PkgVer properties across these two .props files right? @reyang

Reply:
Right, the nonprod file has already included the Common.props so there shouldn't be dup.

https://github.com/open-telemetry/opentelemetry-dotnet/blob/bf57502ae31c16d02987f86944df961ae8118479/build/Common.nonprod.props#L2